app/controllers/api/v1/group_members_controller.rb

```diff
@@ -17,7 +17,14 @@ def index
 
   # POST /api/v1/groups/:group_id/members/
   def create
-    mails_handler = MailsHandler.new(params[:emails], @group, current_user)
+    dummy = GroupMember.new
+    dummy.group_id = params[:group_id]
+    authorize dummy, :primary_mentor?
```

Comment:
```suggestion
    group = Group.find(:group_id)
    authorize group.members.new, :primary_mentor?
```

Reply:
You can do something like this, but what are we testing here ?

Reply:
@tachyons using `group.members.new`, will it assign `group_id` to the new object itself?
also should it not be
```
    group = Group.find(params[:group_id])
    authorize group.members.new, :primary_mentor?
```

Reply:
yes

Reply:
@vedant-jain03 Could you make this change as well?